test/test.js

```diff
@@ -0,0 +1,128 @@
+// node-packages, requires mocha
```

Comment:
Created test functions to ensure underscore -> lodash preserves functionality & recreated rm/rmdir error with previous change